### DISCUSSION TITLE
Rider designer?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [alternetsoft/AlternetUI#169] [Ideas]
Rider now has a free community version.
Can we get a designer plugin for AlterNET UI?

Steps to do so might be.
1. Creation of uixml files in editor as well as thier backing classes
2. Visual ui display for uixml
3. Event connecting and disconnecting from editor
4. Drag and drop ui elements
5. Data Binding

You should take notes from the winforms designer.

**Comment by generalloki** [comment]
I believe we will make a decision about the Rider https://www.jetbrains.com/rider/
support after Alternet.UI Version 1.0 is released. There are plenty of more important things to do.
